sdk/storage/blobs/inc/az_storage_blobs.h

```diff
@@ -42,6 +56,19 @@ typedef struct
   } _internal;
 } az_storage_blobs_blob_client;
 
+
+/**
+ * @brief Init a client with default options
+ *
+ * @param client blob_client to initialize
```

Comment:
Capitalize and add a period.
```suggestion
 * @param client The blob client instance to initialize.
```